src/main/java/uk/gov/hmcts/ccd/endpoint/std/DraftsEndpoint.java

```diff
@@ -71,4 +90,37 @@ public DraftResponse updateDraftForCaseWorker(
         return upsertDraftOperation.executeUpdate(uid, jurisdictionId, caseTypeId, eventTriggerId, draftId, caseDataContent);
     }
 
+    @Transactional
+    @RequestMapping(value = "/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/drafts/{did}",
+        method = RequestMethod.GET)
+    @ApiOperation(value = "Fetch a draft for display")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "A displayable draft")
+    })
+    public CaseView findDraft(@PathVariable("jid") final String jurisdictionId,
+                              @PathVariable("ctid") final String caseTypeId,
+                              @PathVariable("did") final String did) {
+        Instant start = Instant.now();
+        CaseView caseView = getDraftViewOperation.execute(jurisdictionId, caseTypeId, did);
+        final Duration between = Duration.between(start, Instant.now());
+        LOG.warn("findDraft has been completed in {} millisecs...", between.toMillis());
+        return caseView;
+    }
+
+    @Transactional
+    @RequestMapping(value = "/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/drafts/{did}",
+        method = RequestMethod.DELETE)
+    @ApiOperation(value = "Delete a given draft")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "A draft deleted successfully")
+    })
+    public void deleteDraft(@PathVariable("jid") final String jurisdictionId,
+                            @PathVariable("ctid") final String caseTypeId,
+                            @PathVariable("did") final String did) {
+        Instant start = Instant.now();
+        draftGateway.delete(did);
+        final Duration between = Duration.between(start, Instant.now());
+        LOG.warn("deleteDraft has been completed in {} millisecs...", between.toMillis());
```

Comment:
That's not a warning, an info at best. And I believe AppInsights already capture the request timing without that so it may not be needed at all.

Reply:
Yeah ok. Fixed now.